contract/api/ctx.go

```diff
@@ -45,6 +45,7 @@ type StreamContext interface {
 	GetOpId() string
 	GetInstanceId() int
 	GetRootPath() string
+	GetRuleWaitGroup() *sync.WaitGroup
```

Comment:
Looks like this is an internal signal, do not need to export here? Just pass into node when creating topo/planning.

Reply:
updated.